README.md

```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://img.shields.io/travis/openebs/openebs/master.svg?style=flat-square)](https://travis-ci.org/openebs/jiva)
 [![Docker Pulls](https://img.shields.io/docker/pulls/openebs/jiva.svg?style=flat-square)](https://hub.docker.com/r/openebs/jiva/)
 [![Releases](https://img.shields.io/github/release/openebs/openebs/all.svg?style=flat-square)](https://github.com/openebs/openebs/releases)
+[![Go Report Card](https://goreportcard.com/badge/github.com/openebs/openebs)](https://goreportcard.com/report/github.com/openebs/openebs)
```

Comment:
This should be: https://goreportcard.com/report/github.com/openebs/maya

The repo will be shown like it is shown in https://github.com/openebs/maya